terraform/main.tf

```diff
@@ -204,6 +211,24 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
   }
 }
 
+module "grants_prepared_dynamodb_table" {
+  source  = "cloudposse/dynamodb/aws"
+  version = "0.32.0"
+  context = module.this.context
+
+  name             = "prepareddata"
+  hash_key         = "grant_id"
+  table_class      = "STANDARD"
+  enable_streams   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+  # enable_point_in_time_recovery  = true
```

Comment:
This should be un-commented (was this causing LocalStack issues?)

Reply:
Yes, I'm noticing similar behavior documented in the later comments in this thread: https://github.com/localstack/localstack/issues/7569 (using latest, this parameter still fails with that `UnknownOperationException` errro)

Reply:
Similar to my recommendation for the `aws_dynamodb_contributor_insights` resource, it might be worth setting this to an input variable that defaults to `true` but can be set to `false` in `local.tfvars`.

Reply:
This is working now. I believe support was rolled into the latest localstack image.